client/src/components/form/form.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useContext } from 'react';
+
+const FormHeader: React.FC<React.HTMLAttributes<HTMLHeadingElement>> = ({ children, ...props }) => {
+  return (
```

Comment:
I'm not sure if I set the linter for this (I'll double check) but I believe in most cases when the only code inside the `{ }` is a `return` statement you forgo the `{ }`.

```
=> <h2 {...props} >{children}</h2>
```

Reply:
Same for all below

Reply:
okay